examples/plot_scalable_poly_kernels.py

```diff
@@ -0,0 +1,96 @@
+"""
```

Comment:
I think that this example would be great improved by adding a few separators and text, to make a "notebook style example". 
See https://sphinx-gallery.github.io/stable/tutorials/index.html

It's just a question of adding a few separators to insert titles, have "prints" display there results next to where there are in the code, and to turn some comments in text blocks.

Reply:
Done!